src/components/EventsTimeline/TimelineEvent.tsx

```diff
@@ -31,10 +34,38 @@ export type TimelineEventItem = {
 type Props = {
   timelineEventItem: TimelineEventItem
   isOpen?: boolean
+  spacingHorizontal: number
 }
 
-const TimelineEvent: React.FC<Props> = ({ timelineEventItem: { type, caseEvents }, isOpen = false }) => (
-  <div role="button" tabIndex={ -1 }>
+type StylingProps = {
+  spacingHorizontal: number
+}
+
+const Div = styled.div<StylingProps>`
+  position: relative;
+  margin-top: 0;
+  margin-bottom: ${ themeSpacing(5) };
+  margin-left: ${ ( { spacingHorizontal = 0 } ) => themeSpacing( -spacingHorizontal ) };
+  margin-right: ${ ( { spacingHorizontal = 0 } ) => themeSpacing( -spacingHorizontal ) };
```

Comment:
Not `margin: 0 ${ ( { spacingHorizontal = 0 } ) => themeSpacing( -spacingHorizontal ) } ${ themeSpacing(5) };`?